layouts/basic/modules/Vtiger/ConditionBuilder/Smtp.tpl

```diff
@@ -0,0 +1,14 @@
+{*<!-- {[The file is published on the basis of YetiForce Public License 3.0 that can be found in the following directory: licenses/LicenseEN.txt or yetiforce.com]} -->*}
+{strip}
+	<div class="tpl-Base-ConditionBuilder-PickList">
```

Comment:
\<!-- tpl-Base-ConditionBuilder-Smtp -->
{include file=\App\Layout::getTemplatePath...'PickList'
\<!-- /tpl-Base-ConditionBuilder-Smtp -->

Reply:
I can not use the check list template template because I otherwise access the smtp value